### PR TITLE
Feature: Add query for account data

### PR DESCRIPTION
**Pull request by lwojtow**

### CODE REVIEW

File: src/tests/conformance/stateful.rs [hunk 1]
```diff
@@ -1,36 +1,25 @@
...
-        .network_id(TESTNET_NETWORK_ID)
-        .validator_token(testnet.setups[0].validator_token.clone())
-        .add_args(vec![
-            "--valid".into(),
-            "--quorum".into(),
-            "1".into(),
-            "--load".into(),
-        ])
+    let mut node = build_stateful_builder(target.path().to_path_buf())
+        .expect("Unable to get stateful builder")
```
Comment (niklaslong) [inline src/tests/conformance/stateful.rs:15]:
```suggestion
        .expect("unable to get stateful builder")
```

File: src/setup/stateful.rs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+use std::{io, path::PathBuf};
```
Comment (niklaslong) [inline src/setup/stateful.rs:1]: This whole module could probably be in `node` instead?

File: src/tools/rpc.rs [hunk 3]
```diff
@@ -37,32 +68,73 @@ fn build_json_request(request: &impl Serialize) -> RequestBuilder {
...
+}
+
+fn build_account_info_request(account: &str) -> RpcRequest<Vec<AccountInfoRequestParams>> {
+    RpcRequest {
+        id: String::from("1"),
+        method: String::from("account_info"),
+        api_version: API_VERSION,
+        params: vec![AccountInfoRequestParams {
+            account: String::from(account),
+            ledger_index: String::from(""),
```
Comment (niklaslong) [inline src/tools/rpc.rs:87]:
```suggestion
            ledger_index: String::new(),
```

File: src/tests/conformance/stateful.rs [hunk 4]
```diff
@@ -1,36 +1,25 @@
...
         .start(false)
         .await
         .expect("Unable to start stateful node");
     wait_for_state("proposing".into()).await;
+    let account_data = wait_for_account_data("rNGknFCRBZguXcPqC63k6xTZnonSe6ZuWt")
+        .await
+        .expect("Unable to get account data");
```
[the remark this answers is not in the document]
Comment (lwojtow) [reply src/tests/conformance/stateful.rs:22]: Done

File: src/setup/stateful.rs (hunk 2 above)
Comment (lwojtow) [reply src/setup/stateful.rs]: Done

**Review by niklaslong — APPROVED** [review]

**Pull request merged**